addins/Cake.CompressionNetStandard.yml

```diff
@@ -0,0 +1,10 @@
+Name: Cake.CompressionNetStandard
```

Comment:
Needs to be investigated, same packages exists as  addins/Cake.Compression

Reply:
Who needs to investigate? You? Me? And in the mean time what do you suggest we do? allow the yaml to be added to Cake's web site? Blacklist this addin?

Reply:
@Jericho for now, I would suggest that this one isn't added to the website.  Ideally, the work that was done to create this addin will be merged into the Cake.Compression library, and this one unlisted.  Leave this one with me to investigate, and for now, don't include it in the site.